recipes/maxit/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -exo pipefail
+
+# Disable parallel build
+export CPU_COUNT=1
+
+export RCSBROOT="${SRC_DIR}"
+
+case "${OSTYPE}" in
+    darwin*)
+        alias sed="${BUILD_PREFIX}/bin/sed"
+        ln -s "${BUILD_PREFIX}/bin/clang" "${BUILD_PREFIX}/bin/gcc"
+        ln -s "${BUILD_PREFIX}/bin/clang++" "${BUILD_PREFIX}/bin/g++"
+        ;;
+
+    linux*)
+        # To pass CI test on amd64 platforms
+        ulimit -v 2097152
+
+        ln -s "${CC}" "${BUILD_PREFIX}/bin/gcc"
+        ln -s "${GXX}" "${BUILD_PREFIX}/bin/g++"
+        ;;
+    *)
+        exit 1
+        ;;
+esac
+
+cd ${SRC_DIR}/maxit-v10.1/src && \
+sed -i "s|rcsbroot = getenv(\"RCSBROOT\")|rcsbroot = \"${RCSBROOT}\"|g" maxit.C process_entry.C generate_assembly_cif_file.C
```

Comment:
```suggestion
sed -i.bak "s|rcsbroot = getenv(\"RCSBROOT\")|rcsbroot = \"${RCSBROOT}\"|g" maxit.C process_entry.C generate_assembly_cif_file.C
```
If you provide an extension to `sed -i` then there is no need to install GNU/Conda sed for OSX. The native (BSD) sed will work the same way.